python/mspasspy/global_history/manager.py

```diff
@@ -0,0 +1,361 @@
+import os
+import yaml
+import pymongo
+import pyspark
+import collections
+import dask.bag as daskbag
+
+from bson.objectid import ObjectId
+from mspasspy.ccore.utility import MsPASSError
+from datetime import datetime
+from dill.source import getsource
+
+import mspasspy.algorithms.signals as signals
+
+def mspass_spark_map(self, func, *args, global_history=None, object_history=False, alg_id=None,
```

Comment:
I found that some of the docstrings are out of sync right now. For example, the `alg_id` is missing here. You probably want to add something like: 
```
:param alg_id: a user specified alg_id for the map operation
:type alg_id: :class:`str`/:class:`bson.objectid.ObjectId`
```
Also, the description doesn't say anything about `alg_id`, which should definitely be added.

Reply:
Yes, I forgot to sync the docstring before. Now it's updated.